lms/djangoapps/email_marketing/tasks.py

```diff
@@ -120,6 +120,11 @@ def update_user(self, sailthru_vars, email, site=None, new_user=False):
                                  max_retries=email_config.sailthru_max_retries)
 
 
+def is_white_label(self, site):
```

Comment:
Also change the name of this function, I think this is not checking this site is white label.